### PR TITLE
Handle async methods

### PR DESCRIPTION
**Pull request by quietbits**
Added suspend methods to make Android happy

### CODE REVIEW

**Review by JakeUrban — COMMENTED** [review]
I haven't reviewed yet, but at a high level, is `suspend` & `runBlocking` andriods way of specifying sync vs. async code?

**Comment by quietbits** [comment]
> I haven't reviewed yet, but at a high level, is `suspend` & `runBlocking` andriods way of specifying sync vs. async code?

Yes, `suspend` notes that the function is async. `runBlocking` is used only for testing. On Android, it's up to the user to decide which scope/thread to use when calling a `suspend` function.

```kotlin
screenScope.launch { // call suspend function here }
```

File: src/test/kotlin/org/stellar/walletsdk/WalletTest.kt [hunk 1]
```diff
@@ -29,7 +30,7 @@ internal class WalletTest {
 
       // Public key
       assertEquals(56, publicKey.length)
-      assertTrue(publicKey.startsWith("G"))
+      assertTrue(publicKey.startsWith(""))
```
Comment (piyalbasu) [inline src/test/kotlin/org/stellar/walletsdk/WalletTest.kt:33]: Jw what the thought here is. Is this placeholder until you have a more inclusive test that includes muxed accts, for example?

**Review by piyalbasu — APPROVED** [review]

File: src/test/kotlin/org/stellar/walletsdk/WalletTest.kt (hunk 1 above)
Comment (quietbits) [reply src/test/kotlin/org/stellar/walletsdk/WalletTest.kt]: That's my mistake, we should have "G" there. Thanks for catching it!

**Review by JakeUrban — APPROVED** [review]

**Pull request merged**